media/js/firefox/accounts.js

```diff
@@ -20,6 +22,11 @@
         fxaIframeSrc = fxaIframeSrc.replace('context=iframe', 'context=fx_firstrun_v2');
     }
 
+    // if email address provided in querystring, send to FxA frame
+    if (params.get('fxa-email')) {
+        fxaIframeSrc += '&email=' + params.get('fxa-email');
```

Comment:
@shane-tomlinson - We're planning to add an email address field to [/firefox/sync/](http://mozilla.org/firefox/sync) for Fx users that are *not* signed in to Sync. If they fill out the field and click the button, they'll be taken to [/firefox/accounts/](https://www.mozilla.org/firefox/accounts/) and have their email address populated in the FxA iframe.